src/components/layout/home/hero/profile-image.tsx

```
@@ -126,33 +121,23 @@ export function ProfileImage() {
-              alt={imageAlt}
-              src={Cyberpunk}
-              className="hidden rounded-lg transition-transform duration-500 ease-(--ease-fluid) translate-y-0 scale-100 transform-gpu group-hover:-translate-y-1 group-hover:scale-105 cyberpunk:block"
-              loading="eager"
-              priority
-              fill
-              sizes="(min-width: 1024px) 500px, 100vw"
-            />
-          </>
-        ) : (
+        aria-busy={mounted && isEnvDrivenVariant && !isImageLoaded}>
+        <style dangerouslySetInnerHTML={{ __html: profileImgCss }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)